compose/ui/ui/src/jsNativeMain/kotlin/androidx/compose/ui/native/ComposeLayer.jsNative.kt

```diff
@@ -63,10 +66,13 @@ internal class ComposeLayer(
         }
 
         override fun onPointerEvent(event: SkikoPointerEvent) {
-            if (supportsMultitouch) {
+            if (event.pointers.firstOrNull()?.device == SkikoPointerDevice.TOUCH) {
+                if (scene.platformContext.inputModeManager.inputMode != InputMode.Touch) {
```

Comment:
Can you please clarify what's the reasoning behind this check? When in theory we'll get a touch event but will be not in touch input mode? I might miss something since I haven't dove into touch events for quite a while, but from first glance this looks like  an attempt to initialise something that may me make sense to initialise/configure somewhere else, but not in the event handler itself. 

@eymar what do you think?

Reply:
We can connect keyboard/mouse to mobile device. And next event should change method from multi to single for hovering as example.